DataFormats/CLHEP/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
 <use   name="rootmath"/>
 <use   name="DataFormats/Math"/>
 <export>
-  <lib   name="1"/>
   <use   name="DataFormats/Math"/>
+  <use   name="clhep"/>
+  <lib   name="1"/>
```

Comment:
@smuzaffar Could you confirm that adding `clhep` into the export section a necessary/useful change?

Reply:
You are right about your concern. I Realized in the meantime that exporting `clhepheader` might be enough. I'll try that out quickly. I'm convinced though `clhep` dependency has to be exported because CLHEP/xyz is included in the headers of DataFormats/CLHEP

Reply:
I was more thinking if the libraries need to be included in the `<export>` section is necessary at all (mostly because of my poor memory and inability to find anything concrete on the subject, but most of our BuildFiles do not contain any `use` elements inside `export`, and still the dependencies carry over).

Reply:
All the dependencies of a package are automatically exported so there is no need to export any dependency via `<export> </export>` section.

`<lib name="1"/>` is still needed otherwise scram things that it is a private library/plugin and will not allow any one to depend on it.

Reply:
Thanks @smuzaffar. Then I'd suggest to shorten the export section to
```xml
<export>
  <lib name="1"/>
</export>
```

Reply:
Or just leave the file as it is and remove the need for core signature :)

Reply:
Thanks a lot for the discussion, good to know that the exports are automatic! I changed my last commit in a way such that the exports in DataFormats are not touched anymore.

Reply:
To compare, that's how the last commit was before: https://github.com/guitargeek/cmssw/commit/493af00e2b7bcdf32d16880aabfdef313d234e84